schema/swagger.yml

```diff
@@ -14,18 +14,21 @@
 swagger: '2.0'
 
 info:
-  description: Lisk API Documentation
   title: Lisk API Documentation
-  version: '1.0.26'
+  description: |
+    Welcome to Lisks API documentation!
+
+    Below you can see all possible API calls, ordered by endpoints.
+
+    By clicking the "Try it out"-button beside each API call, you can test out the API calls and their responses interactively.
```

Comment:
We need to update this section further: 

1. Since we don't have that featured enabled and planned for now, so we should remove this line. 
2. Add more details are here, which audience is target for this REST API
3. Small summary on what users can do with current available endpoints 
3. A clearly highlighted notice that current version of API have breaking changes with previous version. 

Out of many see these few examples 

http://api.population.io
https://api.carnivalmobile.com

How they explained the API documentation and provided a bit of detail information on usage of it. Like time formats used in API, some detail about authentication etc.